src/functions-and-arrays.js

```diff
@@ -52,14 +131,35 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
+function uniquifyArray(uniqueArray) {
+  if (uniqueArray.length === 0) {
+    return null
+  }
+  const result = []
+  for (let i = 0; i < uniqueArray.length; i++) {
+    if (!result.includes(uniqueArray[i])) {
+      result.push(uniqueArray[i])
+    }
+  }
+  return result
+}
```

Comment:
Bien resuelto. Nada más comenzar no se espera que encontréis una solución compleja a esta kata pero la forma más elegante (puedes investigarlo) de resolver este problema sería a través de un Set: https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Global_Objects/Set

Un Set es una estructura de datos que alberga elementos ÚNICOS no repetidos.

Reply:
Perfecto Daniel, gracias por las correcciones :D